lms/djangoapps/bulk_email/models.py

```diff
@@ -70,7 +70,33 @@ class Meta(object):
         app_label = "bulk_email"
```

Comment:
Scout rule: You should remove the note in the module-level docstring that we're using migrations.  That's universally true across the code base, and doesn't need to be called out here.